src/lib.rs

```diff
@@ -1098,6 +1098,21 @@ mod test {
     }
 
     #[test]
+    #[should_panic(expected = "Unexpected value")]
```

Comment:
Sorry for confusion, 
After my last commit I am asserting 'page_timings_result.is_err()'
You can see this change in 'test_page_timings_float' Test